eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -214,28 +217,6 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
             UserDefaults.standard.setValue(request.url?.absoluteString, forKey: "serverURL")
             UserDefaults.standard.setValue(true, forKey: "wasConnectedBefore")
         }
-        /*
-        Open request for external link (asked by user not automatic request for external link) in a new windows (Preview Controller)
-        - WKNavigationType of a automatic request is always = .Others
-        */
-        if (request.url?.absoluteString.range(of: serverDomain!) == nil && navigationAction.navigationType != WKNavigationType.other) {
```

Comment:
This code permits to open external links in a new window. I think we will have to keep it unless requested by our PO.
This code also fixes the problem of SAML, could you make sure that the SAML fix remains ?

Reply:
Yes, it's okay

Reply:
It seems this check does not take care of SAML requests , isn't it ?

Reply:
No it's already handled, in the same webview , just the call to the external link was fixed here.